README.md

```
@@ -51,6 +52,97 @@
+</td> 
+</tr>
+</table>
+
+## 🏗 System Architecture
+
+### 📌 ER Diagram  
+
+<img src="public/images/ER-Diagram.png" alt="ER Diagram" width="100%">
+
+
+### 📊 Dashboards  
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)